Resources/Prototypes/_Backmen/Entities/Objects/Consumable/Food/Baked/donut.yml

```
@@ -0,0 +1,179 @@
+        - ReagentId: Nutriment
+          Quantity: 10
+
+- type: entity
+  name: Jelly slug hunter
+  parent: FoodDonutBase
+  id: JellySlugHunter
+  description: Seems rotting.
+  components:
+  - type: FlavorProfile
+    flavors:
+      - gfdgdglfdjgjlsjfg;ldg
```

YAMLlint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)